task1-weather/weather.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import json
+import datetime
+import pandas as pd
+
+
+# get time in right format for next three days
+current_day = datetime.date.today()
+end_date = current_day + datetime.timedelta(days=3)
+
+current_day = current_day.isoformat()
+end_date = end_date.isoformat()
```

Comment:
If possible try to avoid assigning a variable another type. Python can deal with it but it is bad practice to avoid confusions throughout the program. You can simply give them a slightly different name.

```suggestion
current_day_str = current_day.isoformat()
end_date_str = end_date.isoformat()
```